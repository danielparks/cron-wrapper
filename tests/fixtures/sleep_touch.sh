#!/bin/bash

# Close stdin, stdout, and stderr so that we test the timeout on wait() rather
# than poll() (used in other fixtures).
exec 0<&- 1<&- 2<&-

sleep 0.1
touch "$1"
