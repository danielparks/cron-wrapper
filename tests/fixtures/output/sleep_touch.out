Command: ["tests/fixtures/sleep_touch.sh"]
Start: 2023-07-26T13:32:23.351057000-07:00

0.117 exit 1
